### PR TITLE
Delay congestion epochs first decrease

### PR DESCRIPTION
**Pull request by Atlante45**
- Delay the first decrease of each congestion epoch to the 2nd NAK if the first NAK is for a single packet.
      This makes the protocol more resilient on networks with small but constant loss,
      and especially networks subjected to a high number of out of orde packets.
- Make sure all stats are in Mb/s, this makes it easier to read and understand the data

### CODE REVIEW

**Comment by Atlante45** [comment]
https://app.asana.com/0/26551373415487/103163614695195

Testing:
- Smoke test asset transfers on different domains

File: tools/udt-test/src/UDTTest.cpp [hunk 1]
```diff
@@ -364,7 +364,11 @@ void UDTTest::handleMessage(std::unique_ptr<Message> message) {
 void UDTTest::sampleStats() {
     static bool first = true;
     static const double USECS_PER_MSEC = 1000.0;
-    
+    static const double MEGABITS_PER_BYTE = 8.0 / 1000000.0;
+    static const double MS_PER_SECOND = 1000.0;
```
Comment (birarda) [inline tools/udt-test/src/UDTTest.cpp]: what do you need MEGABITS_PER_BYTE and MS_PER_SECOND FOR if you're not using either right below?

**Comment by hifi-gustavo** [comment]
Build console output for commit: 31942d9a73eaa824fa088ada6fa7b162971a0f88 available at: https://gist.github.com/320518e8b038a3b247de9984efb4fc76<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7532.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7532.dmg

File: tools/udt-test/src/UDTTest.cpp (hunk 1 above)
Comment (Atlante45) [inline tools/udt-test/src/UDTTest.cpp]: Whoops, forgot to use `MEGABITS_PER_BYTE`.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 0409479a12f3e4835dddf15bc01615267bd12005 available at: https://gist.github.com/320518e8b038a3b247de9984efb4fc76<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7532.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7532.dmg

**Comment by OmegaHeron** [comment]
Tested for proper stack side operation on Linux and client side on windows 7 - no issues seen, but, it really needs Smokey test.

**Pull request merged**